CMakeLists.txt

```
@@ -68,11 +68,20 @@ endif()
 find_package(VulkanHeaders REQUIRED CONFIG)
 
 # Find and create the Vulkan::Vulkan library
-find_library(Vulkan_LIBRARY NAMES vulkan vulkan-1)
-add_library(Vulkan::Vulkan UNKNOWN IMPORTED)
-set_target_properties(Vulkan::Vulkan PROPERTIES
-    IMPORTED_LOCATION ${Vulkan_LIBRARY})
+add_library(Vulkan::Vulkan SHARED IMPORTED)
 target_link_libraries(Vulkan::Vulkan INTERFACE Vulkan::Headers)
+if (WIN32)
+    find_library(Vulkan_LIBRARY_DLL NAMES vulkan-1.dll PATH_SUFFIXES bin)
+    find_library(Vulkan_LIBRARY_IMPLIB NAMES vulkan-1 PATH_SUFFIXES lib)
```

The PATH_SUFFIXES is not needed here. `lib` is already a default suffix. `bin` isn't.

```suggestion
    find_library(Vulkan_LIBRARY_IMPLIB NAMES vulkan-1.lib)
```